src/main.ts

```diff
@@ -169,19 +172,23 @@ async function getBrowser() {
     });  
   }
 }
+
 async function simulate(rawLevelUrl:string, tickRate:number, drawModulo:number, folder:string, shouldRecord:boolean) {
   var cxt = await loadGame(rawLevelUrl, tickRate, drawModulo, folder);
 
   try {
-    return await executeGame(cxt, shouldRecord)
-
+    const result = await executeGame(cxt, shouldRecord)
+    metrics.increment("game.simulate.success", 1);
```

Comment:
Should this factor in the 'result' state?

Reply:
I think the current counter here reporting success is fine, but it'd be nice to have a custom metric for the 'time' - which I believe in some cases can be infinite.  For cases where it's non-infinite, it'd be nice to emit a timer metric (this would represent the total sinerider simulation time before victory.  Where it's infinite, it'd be nice to emit a counter metric denoting "game timeouts", i.e. solutions that didn't complete in under 30s